v1/anbox-cloud-appliance.yaml

```diff
@@ -0,0 +1,21 @@
+description: Anbox Cloud Appliance
```

Comment:
Is it possible to have a more lengthy, multi line description here or is this really just a summary?

Reply:
The primary reason I am asking is that for the appliance to really work after creating the VM we need a valid UA Apps subscription setup. So just launching a VM via `multipass launch anbox-cloud-appliance` will not makes things work directly. Having a description to explain give hints would be useful.

Reply:
Yeah, understood completely, we'll think of a `multipass info <workflow>` of some kind.

It may make sense to display it after `launch`, too.

Reply:
I was even thinking to have it before `launch` so the user doesn't spend time in waiting until the VM is up to only learn that he cannot use it as he's missing a subscription he currently cannot easily get (as soon as the shop allows buying one, he can).